### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by acbecv** [donggaomu/CFDHI-Net#2]
Hello, I saw that the train_images is 1176. May I ask if you trained by combining the training set and the validation set, and then directly evaluated using the testing set?
